src/main/java/org/apache/commons/beanutils2/ConvertUtilsBean.java

```diff
@@ -516,10 +534,13 @@ private void registerStandard(final boolean throwException, final boolean defaul
     private void registerOther(final boolean throwException) {
         // @formatter:off
         register(Class.class,          throwException ? new ClassConverter()          : new ClassConverter(null));
+        register(ColorConverter.class, throwException ? new ColorConverter()          : new ColorConverter(null));
         register(Enum.class,           throwException ? new EnumConverter()           : new EnumConverter(null));
```

Comment:
Isn't the first argument supposed to be Color? If so, how are the tests passing?